### ISSUE TITLE
[a11y demo] make textarea accessible

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-RespEq#57]
Via stevefaulkner on a11y slackers. the textarea needs associated label `<label for="input">Enter TeX code here:</label>`

Sigh.

**Comment by zorkow** [comment]
Done.
